packages/backend/src/peripherals/database/EventRepository.ts

```diff
@@ -72,6 +76,38 @@ export class EventRepository extends BaseRepository {
     return rows.map(toRecord)
   }
 
+  async getAggregatedByProjectAndGranularity(
+    projectId: ProjectId,
+    granularity: 'hour' | 'day',
+    from: UnixTime = new UnixTime(0),
+  ): Promise<EventRecordAggregated[]> {
+    const knex = await this.knex()
+    const rows = (await knex('events')
```

Comment:
@sz-piotr @michalsidzej how do we want to approach such, more complex, queries in knex? (i know we have similar queries in tx counting feature). 

I would suggest going with raw SQLs and just prevent SQL injections with some kind of knex helper. I don't see any value in dealing with the complexities of knex API.

Reply:
I agree. If the PR is complex enough so that we lose static typing from knex then it probably should be just raw SQL